dandischema/models.py

```diff
@@ -1133,42 +952,57 @@ class BareAssetMeta(CommonModel):
         "nskey": "dandi",
     }
 
+    @validator("digest")
+    def digest_etag(cls, values):
+        print(values)
+        try:
+            if len(values[DigestType.dandi_etag]) != 32:
+                raise ValueError
+        except (KeyError, ValueError):
+            raise ValueError("Digest must have an appropriate dandi-etag value.")
```

Comment:
```suggestion
            raise ValueError(f"Digest must have an appropriate dandi-etag value. Got {values[DigestType.dandi_etag]}")
```
what can I say -- I am a friend of mortals ;-) (note: might need linting)

I assume that the `@validator` would wrap those nicely into `pydantic.ValidationError`s?